src/adapters/middleware-adapter.ts

```diff
@@ -57,8 +57,14 @@ export async function handler(event: CloudFrontRequestEvent): Promise<CloudFront
     return request;
   }
 
+  // In the case where `middleware.ts` response w/ a body, send that back to the client
+  let responseBody
+  const state = Object.getOwnPropertySymbols(response).find(s => s.toString() === 'Symbol(state)')
```

Comment:
is this symbol not exported?

Reply:
For some reason, the symbol wasn't create via `Symbol.for('state')` so I couldn't access it by key.

Reply:
turns out it was much simpler to use `response.text()` to get the body.